gui/guihtml/src/TGHtmlParse.cxx

```diff
@@ -1144,7 +1142,8 @@ TGHtmlElement *TGHtml::InsertToken(TGHtmlElement *pToken,
       if (pElem == 0) return 0;
       if (zArgs) {
          // coverity[secure_coding]
-         strcpy (((TGHtmlTextElement *)pElem)->fZText, zArgs);
+         strlcpy (((TGHtmlTextElement *)pElem)->fZText, zArgs,
+                  sizeof(((TGHtmlTextElement *)pElem)->fZText));
```

Comment:
Here wrong. 
You need to add `fZTextSize` parameter into `TGHtmlTextElement` class and assign it in the constructor.
And use it here:
```
         strlcpy (((TGHtmlTextElement *)pElem)->fZText, zArgs,
                  sizeof(((TGHtmlTextElement *)pElem)->fZTextSize));
```

Reply:
As previous comments, I will add `// NOLINT`